indigo_api/views/documents.py

```diff
@@ -122,6 +122,7 @@ def content(self, request, *args, **kwargs):
 
         if request.method == 'PUT':
             try:
+                # TODO: don't just reset here, integrate the content IFF in provision mode (or always?)
```

Comment:
how do I hit this breakpoint?

Reply:
I don't think this is called at all any more; it can be removed (and the `PUT` above) and we'll see when it breaks.